converters/toZigbee.js

```diff
@@ -59,6 +59,20 @@ const converters = {
             await entity.read('genOnOff', ['startUpOnOff']);
         },
     },
+    moes_power_on_behavior: {
+        key: ['moes_power_on_behavior'],
```

Comment:
Please name it `power_on_behavior` instead of `moes_power_on_behavior`

Reply:
But there is already power_on_behavior, with  "const lookup = {'off': 0, 'on': 1, 'toggle': 2, 'previous': 255};"
And for Moes  MS-104BZ you needs"const lookup = {'off': 0, 'on': 1, 'previous': 2};"

Reply:
I know, I didn't mean changing the name of the converter but they `key` and property in `fromZigbee`: https://github.com/Koenkk/zigbee-herdsman-converters/pull/2149/files#diff-b676ab0df4d49b8572967cb4c92a2dc6264091131f6b1828ca13d0f9656e367aR657